modules/social_features/social_activity/modules/social_activity_filter/social_activity_filter.install

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for social_activity_filter module.
+ */
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function social_activity_filter_update_dependencies() {
+  $dependencies['social_activity_filter'][8002] = [
+    'social_core' => 8805,
+  ];
+  return $dependencies;
+}
+
+/**
+ * Implements hook_install().
+ *
+ * Perform actions related to the installation of social_activity_filter module.
+ */
+function social_activity_filter_install() {
+  user_role_grant_permissions('sitemanager', ['administer social_activity_filter']);
+  social_activity_filter_update_8001();
```

Comment:
Please don't invoke update hooks in a install.

Update hooks are there to update existing installations, if you want to do something on install please use the `config/install` files, or create a separate method which you can invoke both from a hook_update_n and the hook_install.

Reply:
ok, I will create a separate method for that